homeassistant/components/zwave/__init__.py

```diff
@@ -387,16 +387,17 @@ def value_added(node, value):
             name = "{}.{}".format(component, object_id(value))
             node_config = hass.data[DATA_DEVICE_CONFIG].get(name)
 
-            if node_config.get(CONF_IGNORED):
-                _LOGGER.info("Ignoring device %s", name)
-                return
-
-            polling_intensity = convert(
-                node_config.get(CONF_POLLING_INTENSITY), int)
-            if polling_intensity:
-                value.enable_poll(polling_intensity)
-            else:
-                value.disable_poll()
+            if node_config:
```

Comment:
Are you sure this helps?
If the yaml section is missing the value of `node_config` should be {} because of `vol.Optional(CONF_DEVICE_CONFIG, default={}):`